app/views/shared/_application_form.erb

```diff
@@ -0,0 +1,5 @@
+<h1 class="govuk-heading-m">Application form</h1>
+<p class="govuk-hint">Do we accept applications from this country?</p>
+<%= f.govuk_radio_button :eligibility_enabled, 'yes', label: { text: 'Yes' }, link_errors: true %>
+<%= f.govuk_radio_button :eligibility_enabled, 'no', label: { text: 'No' } %>
```

Comment:
```suggestion
<%= f.govuk_collection_radio_buttons :eligibility_enabled,
                                     [
                                       OpenStruct.new(value: :false, label: "Yes"),
                                       OpenStruct.new(value: :true, label: "No")
                                     ],
                                     :value,
                                     :label,
                                     legend: { text: "Do we accept applications from this country?" } %>
```

For radio buttons, we should use the `govuk_collection_radio_buttons` method which outputs HTML with a `label` element and links the radio buttons together for accessibility.

https://govuk-form-builder.netlify.app/form-elements/radios/#radios-collection-with-legend-and-hint-text